crates/server/src/admin/service.rs

```diff
@@ -118,6 +120,14 @@ pub(crate) fn setup(
             "/add-client-key",
             post(handlers::add_client_key::add_client_key_handler),
         )
+        .route(
+            "/refresh-jwt-token",
```

Comment:
What happens if your send random values or valid jwt but not created in this node? 
Example we both running nodes and created a JWT. And I call refresh on your node with my JWT (I somehow know where your node is running)